packages/eslint-plugin-shopify/lib/config/rules/lodash.js

```diff
@@ -0,0 +1,84 @@
+// see https://github.com/wix/eslint-plugin-lodash
+
+module.exports = {
+  // Possible errors
+
+  // Use or avoid thisArg for Lodash method callbacks, depending on major version.
+  'callback-binding': 'warn',
+  // Use value returned from collection methods properly.
+  'collection-method-value': 'warn',
+  // Always return a value in iteratees of Lodash collection methods that aren't forEach.
+  'collection-return': 'error',
+  // Do not use .value() on chains that have already ended (e.g. with max() or reduce()) (fixable)
+  'no-double-unwrap': 'error',
+  // Do not use superfluous arguments on Lodash methods with a specified arity.
+  'no-extra-args': 'error',
+  // Prevent chaining without evaluation via value() or non-chainable methods like max().
+  'unwrap': 'error',
```

Comment:
Ehh probably not. I like chaining in theory, but I avoided it with coffeescript because it looks pretty gross with coffeescript-style brace-free functions. I don't think there's much of it right now and if we find this too restricting we can always turn it off